Cargo.toml

```diff
@@ -128,7 +128,7 @@ perfcnt = "0.8.0"
 pest = "2.5.6"
 pest_derive = "2.5.6"
 pretty_assertions = "1.3.0" # update roc_std/Cargo.toml on change
-proc-macro2 = "1.0.51"
+proc-macro2 = "1.0.63"
```

Comment:
the nightly compiler (required for miri, but also useful for lots of other things) does not build with the older version.